common/test/acceptance/tests/helpers.py

```diff
@@ -277,8 +277,8 @@ class EventsTestMixin(object):
     def setUp(self):
         super(EventsTestMixin, self).setUp()
         self.event_collection = MongoClient()["test"]["events"]
-        self.event_collection.drop()
-        self.start_time = datetime.now()
+        self.browser_event_collection = MongoClient()["test"]["browser_events"]
```

Comment:
Is it necessary to have a separate collection now that we resolved the core issue?